students/julimatos.Rmd

```diff
@@ -0,0 +1,43 @@
+##### NEED TO FIX IMAGE, AND READ CSV
```

Comment:
The first 3 lines need to be deleted, such that the very first line is ---. This is so when you go to Knit HTML, the front matter (in between --- and ---) is properly interpreted.